tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -63,14 +63,24 @@ This tutorial has been written to work with Ubuntu 16.04 LTS (Xenial Xerus) and
 
 ### Installing dependencies
 
-We will now update the APT database and install snapcraft and some additional build tools we are
-going to need:
+We will now update the APT database and install some additional build tools we are going to need:
 
 ```bash
 sudo apt update
-sudo apt install snapcraft build-essential
+sudo apt install build-essential
 ```
 
+We will now install snapcraft, the utility for building snaps:
+
+```bash
+sudo snap install --classic snapcraft
+```
+
+positive
```

Comment:
It that some kind of a build or note instruction? :D

Reply:
If you mean the usage of the positive/negative terms, refer [Admonitions](https://github.com/canonical-websites/tutorials.ubuntu.com/blob/master/examples/example-tutorial.md#admonitions)

Reply:
Thanks. Admonitions could use more specific markup to be more obvious.